### ISSUE TITLE
accessibility: adapt-contrib-glossay general fixes

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2199]
### Subject of the issue
* Search input label doesn't read
* Clear button not accessible
* List isn't reading as list
* aria-expanded not working
* Search result doesn't alert

**Comment by oliverfoster** [comment]
merged
